2021-06-18 04:01:12,133 INFO    MainThread:4031805 [internal.py:wandb_internal():91] W&B internal server running at pid: 4031805, started at: 2021-06-18 04:01:12.132813
2021-06-18 04:01:12,139 DEBUG   HandlerThread:4031805 [handler.py:handle_request():124] handle_request: run_start
2021-06-18 04:01:12,141 INFO    WriterThread:4031805 [datastore.py:open_for_write():80] open: /home/chengzilong/DDQN/wandb/offline-run-20210618_040110-3bvy50pl/run-3bvy50pl.wandb
2021-06-18 04:01:14,347 DEBUG   HandlerThread:4031805 [meta.py:__init__():39] meta init
2021-06-18 04:01:14,347 DEBUG   HandlerThread:4031805 [meta.py:__init__():53] meta init done
2021-06-18 04:01:14,347 DEBUG   HandlerThread:4031805 [meta.py:probe():210] probe
2021-06-18 04:01:14,354 DEBUG   HandlerThread:4031805 [git.py:repo():33] git repository is invalid
2021-06-18 04:01:14,354 DEBUG   HandlerThread:4031805 [meta.py:_save_pip():57] save pip
2021-06-18 04:01:14,356 DEBUG   HandlerThread:4031805 [meta.py:_save_pip():71] save pip done
2021-06-18 04:01:14,357 DEBUG   HandlerThread:4031805 [meta.py:_save_conda():78] save conda
2021-06-18 04:01:18,527 DEBUG   HandlerThread:4031805 [meta.py:_save_conda():86] save conda done
2021-06-18 04:01:18,528 DEBUG   HandlerThread:4031805 [meta.py:probe():252] probe done
